### DISCUSSION TITLE
What is the line/border between my window bar and i3bar?

### DISCUSSION THREAD

**Discussion by c02y** [i3/i3#4601] [Q&A] [answered]
I use https://thomashunter.name/i3-configurator/ to configure the color in i3bar and windows
```
# class                 border  bground text    indicator child_border
client.focused          #285577 #285577 #FFFFFF #2E9EF4   #285577
client.focused_inactive #333333 #5F676A #FFFFFF #484E50   #5F676A
client.unfocused        #333333 #222222 #888888 #292D2E   #222222
client.urgent           #2F343A #900000 #FFFFFF #900000   #900000
client.placeholder      #000000 #0C0C0C #FFFFFF #000000   #0C0C0C

client.background       #FFFFFF

bar {
  colors {
    background #000000
    statusline #FFFFFF
    separator  #666666

    focused_workspace  #285577 #285577 #FFFFFF
    active_workspace   #333333 #222222 #FFFFFF
    inactive_workspace #333333 #222222 #888888
    urgent_workspace   #2F343A #900000 #FFFFFF
    binding_mode       #2F343A #900000 #FFFFFF
  }
}
```
The above config gets the following result:
![image](https://user-images.githubusercontent.com/2135996/137693896-8b258efa-8233-4eb6-85cd-93e1f783cf6c.png)

I paste the above config into my config and reload, I get:
![image](https://user-images.githubusercontent.com/2135996/137694028-c9906236-d6b2-4582-b26e-5cfb85057d49.png)

You can see there is a black line/border between the i3bar and the window bar.

How can I get rid of that line?

**Comment by Airblader** [comment, marked as answer]
The workspace buttons in i3bar are inset by one pixel, so that black line is actually the background of i3bar. There is no configuration to remove this 1px padding, you can only change the color of i3bar's background.

**Reply by c02y** [reply]
I don't think it is the issue of the workspace buttons, there is a line between the whole window bar and the whole i3bar. 

No plans to make it an option?

I notice that the line is black, I'm trying to make the whole part black which will be the same color of the line so I won't see that line.

**Reply by Airblader** [reply]
> No plans to make it an option?

We do not plan on changing it, no.

**Reply by c02y** [reply]
OK, thanks.

See, much better:
![image](https://user-images.githubusercontent.com/2135996/137703845-13ffe009-e73f-4a94-bf9d-9b7433d51ead.png)

Can I set the color for the line? Seems neither. Making it pure black is not a really good choice.
